tools/extras/install_srilm.sh

```diff
@@ -24,7 +24,7 @@ tar -xvzf ../srilm.tgz
 # set the SRILM variable in the top-level Makefile to this directory.
 cp Makefile tmpf
 
-cat tmpf | awk -v pwd=`pwd` '/SRILM =/{printf("SRILM = %s\n", pwd); next;} {print;}' \
+cat tmpf | gawk -v pwd=`pwd` '/SRILM =/{printf("SRILM = %s\n", pwd); next;} {print;}' \
```

Comment:
Then I am going to fix another line in this script. Wait for the PR to be updated before merging.